recipe/meta.yaml

```diff
@@ -17,18 +17,21 @@ build:
 
 requirements:
   build:
-    - python
+    - python >=3.5
```

Comment:
So either this should be used and the `skip` dropped or the `skip` should be refreshed to match this constraint. If the former, this will also need to be `noarch` as well. Should add combining both of these and re-rendering caused the CIs to get disabled.

Reply:
So the best approach should be just use `python` (with no version pinned) and add the selector on skip?

Reply:
Depends on whether you would like to make this package `noarch` or not.

Reply:
Will noarch work with skip true for py2k?